docs/Install.md

```diff
@@ -0,0 +1,56 @@
+# Install
+
+Inductiva package is simple to install, just run on your terminal:
```

Comment:
Maybe we want to have the install info on the top page, and the trouble shooting info on this separate page.